xbmc/filesystem/CurlFile.cpp

```diff
@@ -540,7 +540,7 @@ void CCurlFile::SetCommonOptions(CReadState* state)
   if (m_postdataset)
   {
     g_curlInterface.easy_setopt(h, CURLOPT_POST, 1 );
-    g_curlInterface.easy_setopt(h, CURLOPT_POSTFIELDSIZE, m_postdata.length());
+    g_curlInterface.easy_setopt(h, CURLOPT_POSTFIELDSIZE, m_postdata.size());
```

Comment:
>Note that string objects handle bytes without knowledge of the encoding that may eventually be used to encode the characters it contains. Therefore, the value returned may not correspond to the actual number of encoded characters in sequences of multi-byte or variable-length characters (such as UTF-8).

See - they do it exactly the same. Can you post a string where this is different? I am really only curious - as I did not see something like that in an std::string until now.